__tests__/app.test.js

```diff
@@ -206,6 +211,138 @@ describe('GET /api/articles/:article_id/comments', () => {
   });
 });
 
+describe('POST /api/articles/:article_id/comments', () => {
+  test('A succesful comment should respond with an appropriate status and the posted comment with correct comment properties', () => {
+    const newComment = {
+      username : 'lurker',
+      body : "body..."
+    };
+    return request(app)
+      .post('/api/articles/1/comments')
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        const comment = response.body.comment;
+        
+        expect(comment).toHaveProperty("comment_id")
+        expect(typeof comment.comment_id).toBe("number")
+
+        expect(comment).toHaveProperty("body")
+        expect(typeof comment.body).toBe("string")
+
+        expect(comment).toHaveProperty("article_id")
+        expect(typeof comment.article_id).toBe("number")
+
+        expect(comment).toHaveProperty("author")
+        expect(typeof comment.author).toBe("string")
+
+        expect(comment).toHaveProperty("votes")
+        expect(typeof comment.votes).toBe("number")
+
+        expect(comment).toHaveProperty("created_at")
+        expect(typeof comment.created_at).toBe("string")
```

Comment:
This test is a good foundation but could be improved in a few ways:
- If you're testing `typeof comment.property...` you don't also need to test `.toHaveProperty`, if it doesn't have the property then the typeof test will fail so you only need that one for each property
- In the case of some of these properties, we should know exactly what they are. For example, the _article_id_ should be the one from the path, `1`. The author should be the one from the comment object, `lurker`. Replacing the `typeof` tests for more specific tests on those properties will make your test more secure